File: 122-best-time-to-buy-and-sell-stock-ii/122-best-time-to-buy-and-sell-stock-ii.py
class Solution:
    def maxProfit(self, prices: List[int]) -> int:
        sum = 0
        for i in range(0 , len(prices)-1,1):
            if(prices[i] < prices[i+1]):
                sum += prices[i+1] - prices[i]
        return sum
                
        
        

File: 122-best-time-to-buy-and-sell-stock-ii/122-best-time-to-buy-and-sell-stock-ii.java
class Solution {
    public int maxProfit(int[] prices) {
        int max = 0;
        for(int i = 1 ; i < prices.length ; i++){
            if(prices[i] > prices[i-1]){
                int max2 = prices[i] - prices[i-1];
                max+=max2;
            }
        }
        return max;
    }
}